app/lib/etl-toolkit/queues/populateRecordsFromIds.js

```diff
@@ -34,36 +34,36 @@ function savePeriodically() {
 
 function processQueueItem(task, callback) {
   count += 1;
-  if (pageParsed(task.id)) {
+  if (recordExists(task.id)) {
     callback(false);
   } else {
     savePeriodically();
     log.info(`Populating ID ${task.id} ${count}/${etlStore.getIds().length}`);
     limiter(hitsPerWorker, () => populateData(task.id), () => callback(true));
   }
 }
+
 function processRetryQueueItem(task, callback) {
   retryCount += 1;
   log.info(`Retrying ID ${task.id} ${retryCount}/${totalRetries}`);
   limiter(hitsPerWorker, () => populateData(task.id), callback);
 }
 
-function writeDoneLog(id, writeLog) {
-  if (writeLog) {
-    log.info(`${id} done`);
-  }
+function writeDoneLog(id) {
+  log.info(`${id} done`);
 }
 
 function addToQueue(ids, q) {
   // remove 'undefined's
   ids.filter(id => id).forEach((id) => {
-    q.push({ id }, writeLog => writeDoneLog(id, writeLog));
+    q.push({ id }, () => writeDoneLog(id));
```

Comment:
Previously the function would pass along the 'writeLog' parameter returned from the callback in the `processQueueItem` function. This is 'true' if a record has been updated, and false if skipped.
This prevents the done long being written for the thousands of records that are not actioned.

In the new implementation the done log is always written, adding a lot of output to the log.

Reply:
Yeah, unintentional consequence. I'll put it back to how it was.